docs/user-guide/services/index.md

```diff
@@ -423,6 +423,44 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
+#### SSL support on AWS
+For partial SSL support on clusters running on AWS, starting with 1.3 two
```

Comment:
If this is 1.3-related only, you should technically be documenting it in the release-1.3 branch. @bgrant0607